rotkehlchen/data_import/manager.py

```diff
@@ -0,0 +1,52 @@
+from pathlib import Path
+from typing import Optional, Tuple
+
+from rotkehlchen.data_import.importers.binance import BinanceImporter
+from rotkehlchen.data_import.importers.bisq_trades import BisqTradesImporter
+from rotkehlchen.data_import.importers.blockfi_trades import BlockfiTradesImporter
+from rotkehlchen.data_import.importers.blockfi_transactions import BlockfiTransactionsImporter
+from rotkehlchen.data_import.importers.cointracking import CointrackingImporter
+from rotkehlchen.data_import.importers.cryptocom import CryptocomImporter
+from rotkehlchen.data_import.importers.nexo import NexoImporter
+from rotkehlchen.data_import.importers.shapeshift_trades import ShapeshiftTradesImporter
+from rotkehlchen.data_import.importers.uphold_transactions import UpholdTransactionsImporter
+from rotkehlchen.data_import.utils import BaseExchangeImporter
+from rotkehlchen.db.dbhandler import DBHandler
+from rotkehlchen.types import DataImportSource
+
+
+class DataImporter:
```

Comment:
This is a `CSVDataImporter`
```suggestion
class CSVDataImporter:
```

Also add a proper class docstring explaining what the role of this class is

Reply:
Done